coordinator/src/dlc_protocol.rs

```diff
@@ -506,15 +524,16 @@ impl DlcProtocolExecutor {
         Ok(())
     }
 
-    /// Completes the rollover dlc protocol as successful and updates the 10101 meta data
-    /// accordingly in a single database transaction.
+    /// Complete the rollover DLC protocol as successful and update the 10101 metadata accordingly,
+    /// in a single database transaction.
```

Comment:
🔧 Would be nice if true 🙂

Reply:
Is it in a single database transaction?

Reply:
I think I understand that you want me to move the new `insert` into the `start_dlc_protocol` method, but your messages are a bit cryptic.

That should be an easy fix though.

Reply:
I was referring to the single database transaction comment, which i thought was not true. But I missed that the db transaction is already started in the calling function. Sorry for the confusion, no need to change anything.